### PR TITLE
OT208-22

### PR DESCRIPTION
**Pull request by acamus79**
Add MemberEntity, MemberRepo

### CODE REVIEW

File: src/main/java/com/alkemy/ong/models/MemberEntity.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    private String image;
+    
+    @Column(nullable = true, length = 255)
+    private String description;
+   
+    @Column(nullable = false)
+    @CreatedDate
+    private LocalDateTime created;
+   
+    private boolean active = Boolean.TRUE;
```
Comment (juan1977lopez) [inline src/main/java/com/alkemy/ong/models/MemberEntity.java:77]: Este campo se debe llamar softDelete

**Review by juan1977lopez — COMMENTED** [review]
Revisar comentarios

File: src/main/java/com/alkemy/ong/models/MemberEntity.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    
+    @Column(nullable = false, length = 80)
+    private String image;
+    
+    @Column(nullable = true, length = 255)
+    private String description;
+   
+    @Column(nullable = false)
+    @CreatedDate
+    private LocalDateTime created;
```
[the remark this answers is not in the document]
Comment (acamus79) [reply src/main/java/com/alkemy/ong/models/MemberEntity.java]: done

File: src/main/java/com/alkemy/ong/models/MemberEntity.java (hunk 1 above)
Comment (acamus79) [reply src/main/java/com/alkemy/ong/models/MemberEntity.java]: done

**Review by juan1977lopez — APPROVED** [review]
Aprobado

**Pull request merged**